src/ApiContext.js

```diff
@@ -0,0 +1,4 @@
+import React from "react";
```

Comment:
What is the reason for having an api context? I'm not understanding the purpose of it.

Reply:
This is so i dont have to create the AlgoquantApi object that we get from our sdk everytime we need to use the API. The context allows me to pass the object through all the components through the provider. If its not, there would be a lot of repeated code and new objects because i would have to create a new one in each component. Or have to do nested props, which  can also cause problems.

Reply:
with the context all i need to do is call it once and it an be passed using the provider